tests/test_schema_id_regex.py

```diff
@@ -0,0 +1,126 @@
+import json
+import unittest
+
+from jsonschema import ValidationError, validate
+
+def create_schema_with_id(schema_id='answer'):
+    """
+    Utility method that loads a JSON schema file and swaps out an answer Id.
+    :param schema_id: The Id to use for the answer.
+    :return: The JSON file with the Id swapped for schema_id
+    """
+    schema_path = "tests/schemas/test_percentage.json"
```

Comment:
We should rename the schema to `test_schema_id_regex.json` and simplify it to it's minimum - removing navigation and empty properties.